### PR TITLE
Adds the modsuit Chameleon Module

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Adds the modsuit Chameleon Module.
In uplink for 10 TC.

## Why It's Good For The Game

This module lets you disguise an *inactive* modsuit as a backpack. Does not work when the modsuit is active.
Good for hiding a modsuit in plain sight, and is better in most cases than plate compression, however can not be printed at RND. This should help with the old "I was able to hide my syndicate hardsuit in my bag" complaint

## Testing

Several rounds of testing as I tried to fuse the modsuit code with cham clothing code

## Changelog
:cl:
add: Adds the modsuit Chameleon Module to uplink for 10 TC. A module using chameleon technology to disguise the suit as another object. Does not work when active
/:cl:

### CODE REVIEW

File: code/datums/uplink_items/uplink_general.dm [hunk 1]
```diff
@@ -636,6 +636,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/mod/module/plate_compression
 	cost = 10
 
+/datum/uplink_item/suits/chameleon_module
+	name = "MODsuit Chameleon Module"
+	desc = "A module using chameleon technology to disguise the suit as another object. Does not work when active"
```
Comment (Burzah) [inline code/datums/uplink_items/uplink_general.dm:641]:
```suggestion
	desc = "A module using chameleon technology to disguise the suit as another object. Does not work when active."
```

File: code/datums/uplink_items/uplink_general.dm [hunk 2]
```diff
@@ -660,6 +660,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/mod/module/plate_compression
 	cost = 10
 
+/datum/uplink_item/suits/chameleon_module
+	name = "MODsuit Chameleon Module"
+	desc = "A module using chameleon technology to disguise the suit as another object. Does not work when active."
```
Comment (lewcc) [inline code/datums/uplink_items/uplink_general.dm:665]: What does "does not work when active" mean here?

**Review by lewcc — COMMENTED** [review]
One thing, looks p good tho

File: code/datums/uplink_items/uplink_general.dm (hunk 2 above)
Comment (lewcc) [inline code/datums/uplink_items/uplink_general.dm:665]: This should clarify it a bit, I think.
```suggestion
	desc = "A module using chameleon technology to disguise an undeployed modsuit as another object. Note: the disguise will not work once the modsuit is deployed."
```

File: code/datums/uplink_items/uplink_general.dm [hunk 3]
```diff
@@ -660,6 +660,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/mod/module/plate_compression
 	cost = 10
 
+/datum/uplink_item/suits/chameleon_module
+	name = "MODsuit Chameleon Module"
+	desc = "A module using chameleon technology to disguise an undeployed modsuit as another object. Note: the disguise will not work once the modsuit is deployed, but will can be toggled again when it is retracted."
```
Comment (Henri215) [inline code/datums/uplink_items/uplink_general.dm:665]:
```suggestion
	desc = "A module using chameleon technology to disguise an undeployed modsuit as another object. Note: the disguise will not work once the modsuit is deployed, but can be toggled again when retracted."
```

**Review by Henri215 — APPROVED** [review]
Lewcs good to me (Lewc, take a lewc at it now!)

**Review by Burzah — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**